doc/build_apps/js_app_bundle.rst

```diff
@@ -133,9 +133,6 @@ It is an object implementing the :js:class:`CCF <CCF>` interface documented belo
 .. js:autoclass:: ProofElement
    :members:
 
-.. js:autoclass:: WrapAlgoParams
```

Comment:
This is now a union `type` instead of `interface`, and sphinx-js doesn't support those.

Reply:
So that can revert now presumably?

Reply:
Hm? Union is the correct type for this. I think it's fine as it is.

Reply:
Sorry, mis-read that first comment, ignore me.